.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Don't think these should have been included?

Reply:
Yes, my mistake. Made the commit through vscode and didn't notice these files were also staged. Thanks!